src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,25 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+import mate.academy.service.BookService;
+import mate.academy.service.BookServiceImpl;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        BookService bookService = new BookServiceImpl(bookDao);
+        Book book = new Book("Test book", BigDecimal.valueOf(10));
+        System.out.println(bookService.create(book));
```

Comment:
According to the task description you should test all BookDao methods, not service